modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -455,10 +455,6 @@ def test_input_3d(self):
                                     "Verify OPENCV_DNN_TEST_DATA_PATH configuration parameter.")
 
         input = np.load(input_file)
-        # we have to expand the shape of input tensor because Python bindings cut 3D tensors to 2D
-        # it should be fixed in future. see : https://github.com/opencv/opencv/issues/19091
-        # please remove `expand_dims` after that
-        input = np.expand_dims(input, axis=3)
```

Comment:
`setInput` has been tested in this one. It was previously hacked to make it passed.

---

I also added `CV_ND` macro for `setParam`. Does it need to be tested? This API is not tested any where though.